primerpedia.js

```diff
@@ -62,4 +62,24 @@ function apiRequest(queryString){
 			}
 		}
 	});
-}
\ No newline at end of file
+}
+
+function getQueryVariable(variable) {
+	// http://stackoverflow.com/questions/2090551/parse-query-string-in-javascript
```

Comment:
Thanks for adding documentation! I'd rather link directly to the [relevant answer](http://stackoverflow.com/a/2091331/266309), though.